### ISSUE TITLE
JetEngine - Integration with Breakdance builder

### ISSUE THREAD

**Issue by miguel-cravidao** [CrocoBlock/suggestions#7180]
I want to suggest JetEngine integration with Breakdance. There are already some people requesting JetEngine features to be added to Breakdance. I'm excluding the widgets for Breakdance; I'm just talking about the JetEngine features and integration with post loop.

Breakdance and Bricks can be real options compared with Elementor if they can manage queries, post loops, and visibility rules with JetEngine. That's the reason why I still work with Elementor at the moment. I know that Bricks already have support, and I'll take a look, but Breakdance still doesn't.

The bottom line is that these builders are much more lightweight, perform better, and the code output is miles away from Elementor. With JetEngine support, I can move to any supported page builder, knowing that 90% of my functionalities are fulfilled.

**Comment by Leadsquad** [comment]
That's would be amazing to get a Breakdance integration
